dataGenerators/seedReviews.js

```diff
@@ -88,6 +88,7 @@ var createReview = function() {
   return review;
```

Comment:
Line 27: I don't think parameter 'max' is necessary because it's globally defined on line 18.